docs/ProductionChecklistForScalarDLAuditor.md

```diff
@@ -0,0 +1,172 @@
+# Production checklist for ScalarDL Auditor
+
+This checklist provides recommendations when deploying ScalarDL Auditor in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDL Auditor on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDL Auditor
+
+The following is a checklist of recommendations when setting up ScalarDL Auditor in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. You can see the [sample configurations](../conf/scalardl-audit-custom-values.yaml) of `podAntiAffinity` for making three pods spread across the worker nodes.
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different availability zones (AZs), you can withstand an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDL Auditor are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDL Auditor pods exist on the worker nodes.
+
+In other words, the following components could run on one worker node:
```

Comment:
Does the wording for this depend on how we word a similar sentence in a different PR?: [related comment](https://github.com/scalar-labs/scalar-kubernetes/pull/213/files#r1382880277)

Reply:
Yes! I will update this sentence after the discussion in another PR is finished!

Reply:
Fixed in 259da4d754e7b551d0ade62bef962476cdc79b6c.